dora/tests/integration/src/test/java/alluxio/client/fs/DoraFileSystemIntegrationTest.java

```diff
@@ -94,9 +100,25 @@ public final class DoraFileSystemIntegrationTest extends BaseIntegrationTest {
   @Rule
   public ExpectedException mThrown = ExpectedException.none();
   private AmazonS3 mS3Client = null;
+  @Rule
+  public TemporaryFolder mFolder = new TemporaryFolder();
 
   @Before
   public void before() throws Exception {
+    File file = mFolder.newFile();
+    try (PrintStream ps = new PrintStream(file)) {
+      ps.println("localhost");
+    }
+    mLocalAlluxioClusterResourceBuilder
+        .setProperty(PropertyKey.DORA_CLIENT_UFS_FALLBACK_ENABLED, false)
+        .setProperty(PropertyKey.WORKER_MEMBERSHIP_MANAGER_TYPE, MembershipType.STATIC)
+        .setProperty(
+            PropertyKey.WORKER_STATIC_MEMBERSHIP_MANAGER_CONFIG_FILE, file.getAbsolutePath())
+        .setProperty(PropertyKey.WORKER_RPC_PORT, 24999)
```

Comment:
It seems strange, but without them will raise "Address already in used" in my local test running.

Reply:
I will fix.